src/rgw/rgw_admin.cc

```diff
@@ -7246,5 +7247,6 @@ int main(int argc, const char **argv)
     }
   }
 
+  rgw::curl::cleanup_curl();
```

Comment:
a lot of the admin commands will do an early return, so we can't rely on this to catch everything - that's why i had to go with raii in https://github.com/ceph/ceph/pull/20694

Reply:
makes sense, I'll update rgw_admin